assets/css/course-editor.scss

```diff
@@ -92,3 +93,50 @@ $primary: #6FCFB2;
 		}
 	}
 }
+
+.sensei-pricing-nudge {
```

Comment:
How do you feel about calling it `promo` instead of `nudge`? Dmitry suggested it in one of the PRs and I've went with that.

Reply:
I agree sounds better, I've changed it everywhere :
[Classname ](https://github.com/Automattic/sensei/pull/4810/files#diff-0cdf94ac1cfb0a1e90adaef288fdb975f07f4c39ff34e0de2003256181b3ec93R11)

[css](https://github.com/Automattic/sensei/pull/4810/files#diff-47520f7e1c5b86d12dd032cfd96e86236dcc5c418968a61cab2f6ebd89f3d8f2R97
[
sidebar element](https://github.com/Automattic/sensei/pull/4810/files#diff-46b5b32718313cd62a309f3e5bb5b44af7fd7d418aa97369ac2e33b8204b7d35R97)